src/Games/NexusMods.Games.RedEngine/RedModDeployTool.cs

```diff
@@ -23,18 +24,25 @@ public RedModDeployTool(ILogger<RedModDeployTool> logger)
     public async Task Execute(Loadout.ReadOnly loadout, CancellationToken cancellationToken)
     {
         var exe = RedModPath.CombineChecked(loadout.InstallationInstance);
+        var deployFolder = RedModDeployFolder.CombineChecked(loadout.InstallationInstance);
+        
+        // RedMod deploys to this folder and freaks out if it doesn't exist, but our synchronizer will
+        // delete it if it's empty, so we need to recreate it here.
+        if (!deployFolder.DirectoryExists())
+            deployFolder.CreateDirectory();
 
         var stdOutBuffer = new StringBuilder();
         var stdErrBuffer = new StringBuilder();
 
         _logger.LogInformation("Running {Program}", exe);
         var result = await Cli.Wrap(exe.ToString())
             .WithArguments("deploy")
-            .WithWorkingDirectory(loadout.InstallationInstance.LocationsRegister[LocationId.Game].ToString())
+            .WithWorkingDirectory(exe.Parent.ToString())
             .WithStandardErrorPipe(PipeTarget.ToStringBuilder(stdErrBuffer))
             .WithStandardOutputPipe(PipeTarget.ToStringBuilder(stdOutBuffer))
-            .ExecuteAsync();
+            .ExecuteAsync(cancellationToken);
         _logger.LogInformation("Finished running {Program}", exe);
+        _logger.LogDebug("RedMod Deploy stdout: {StdOut}", stdOutBuffer.ToString());
```

Comment:
You should use `IProcessFactory` for anything process related. Printing the output to the logs is also not necessary, since the process factory does all that for you.